src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -17,7 +17,8 @@
   <ItemGroup>
     <PackageReference Include="Azure.Core" Version="1.18.0-alpha.20210811.1" />
     <PackageReference Include="Azure.Core.Experimental" Version="0.1.0-preview.12" />
-    <PackageReference Include="Azure.ResourceManager" Version="1.0.0-alpha.20210811.3" />
+    <!-- <PackageReference Include="Azure.ResourceManager" Version="1.0.0-alpha.20210811.3" /> -->
+    <ProjectReference Include="../../../azure-sdk-for-net/sdk/resourcemanager/Azure.ResourceManager/src/Azure.ResourceManager.csproj" />
```

Comment:
hope this is temporary.

Reply:
Yes, once https://github.com/Azure/azure-sdk-for-net/pull/23091 is merged and the new version of Azure.ResourceManager is published, will use that new version.